### PR TITLE
Create .gitmessage

### PR DESCRIPTION
**Pull request by c0nn3r**
I really appreciate emojis to provide semantic signals for commit messages.

### CODE REVIEW

**Comment by leijurv** [comment]
Wow powwows wowowowo

The **LEGENDARY** c0nn3r making such a BASED pr to baritone?! i can scarcely believe my eyes

**Comment by 0-x-2-2** [comment]
👎

**Comment by nm17** [comment]
It's about time you show up @c0nn3r

File: .gitmessage [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<emoji> <title> (<ticket>)
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# 📝 Update README.md (WD-1234)
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+# ✅ Add unit test for inputs (WD-1234)
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+# <emoji> can be:
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+# 🎨 :art: when improving structure of the code
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+# ⚡️ :zap: when improving performance
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+# 🔥 :fire: when removing code or files
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+# ✨ :sparkles: when introducing new features
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+# 🚧 :construction: when work in progress
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+# 🔨 :hammer: when refactoring code
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+# 📝 :memo: when writing docs
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+# 💄 :lipstick: when updating the UI and style files
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+# 📈 :chart_with_upwards_trend: when adding analytics or tracking code
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+# 🌐 :globe_with_meridians: when adding internationalization and localization
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+# ✏️ :pencil2: when fixing typos
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 18]
```diff
@@ -0,0 +1,31 @@
...
+# 🚚 :truck: when moving or renaming files
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+# ✅ :white_check_mark: when adding tests
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 20]
```diff
@@ -0,0 +1,31 @@
...
+
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 21]
```diff
@@ -0,0 +1,31 @@
...
+# 👌 :ok_hand: when updating code due to code review changes
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 22]
```diff
@@ -0,0 +1,31 @@
...
+# 🐛 :bug: when fixing a bug
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 23]
```diff
@@ -0,0 +1,31 @@
...
+# 🚑 :ambulance: when doing a critical hotfix
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 24]
```diff
@@ -0,0 +1,31 @@
...
+# 🚨 :rotating_light: when removing linter warnings
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 25]
```diff
@@ -0,0 +1,31 @@
...
+
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 26]
```diff
@@ -0,0 +1,31 @@
...
+# 🔀 :twisted_rightwards_arrows: when merging branches
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 27]
```diff
@@ -0,0 +1,31 @@
...
+# ⬆️ :arrow_up: when upgrading dependencies
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 28]
```diff
@@ -0,0 +1,31 @@
...
+# ⬇️ :arrow_down: when downgrading dependencies
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 29]
```diff
@@ -0,0 +1,31 @@
...
+# 🔧 :wrench: when changing configuration files
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 30]
```diff
@@ -0,0 +1,31 @@
...
+# 🔖 :bookmark: when releasing / version tagging
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

File: .gitmessage [hunk 31]
```diff
@@ -0,0 +1,31 @@
...
+# 💚 :green_heart: when fixing the CI build
```
Comment (0-x-2-2) [inline .gitmessage]:
```suggestion
```

**Comment by cookiedragon234** [comment]
Thanks so much for the suggestions @0x22

**Comment by leijurv** [comment]
Best PR ever

**Comment by leijurv** [comment]
whatever it takes to make cda stop saying `[Change]` in front of every change

emojis instead is a small price to pay for salvation

**Comment by dominikaaaa** [comment]
# :heart_eyes_cat:

**Pull request merged**